generate_report.py

```diff
@@ -0,0 +1,46 @@
+from drift import info_parser
+from drift.app import create_app
+import json
+
+from kerlescan.inventory_service_interface import interleave_systems_and_profiles
+
+print("Testing report generation...")
+# TODO: remove need to initialize flask app
+app = create_app()
+app.app.app_context().push()
+
+systems, profiles, baselines, hsps = [], [], [], []
+with open("tests/reports/inputs/inventory/report1-systems.json", "r") as myfile:
+    systems = json.loads(myfile.read())["results"]
+
+with open("tests/reports/inputs/inventory/report1-system_profiles.json", "r") as myfile:
+    profiles = json.loads(myfile.read())["results"]
+
+systems_with_profiles = interleave_systems_and_profiles(systems, profiles)
+
+with open("tests/reports/inputs/inventory/report1-system_profiles.json", "r") as myfile:
```

Comment:
This is a duplicate of lines 16-17 above.

Reply:
oops!